target/linux/ramips/dts/mt7621_dlink_dir-8xx-a1.dtsi

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	aliases {
+		led-boot = &led_power_orange;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_net_orange;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_orange: power_orange {
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_green: power_green {
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		led_net_orange: net_orange {
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		led_net_green: net_green {
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+	};
```

Comment:
To be honest, since it's only one additional line, I'd personally prefer to keep the whole LEDs block in the DTS files. It would be much easier to read.